### ISSUE TITLE
Error, when visualizing function for the first time

### ISSUE THREAD

**Issue by felipejoribeiro** [ray-x/lsp_signature.nvim#223]
When visualizing a function for the first time, this error occurs:

![image](https://user-images.githubusercontent.com/39244876/197035100-51a79082-f852-4225-a620-5adeef09f7c2.png)

And after this error, the window stays frozen in the screen, and i have to click in it and exit to close it:

https://user-images.githubusercontent.com/39244876/197036136-b8c9c4ec-7575-4c35-bda1-e1fad9ada375.mov

My nvim configuration:

https://github.com/felipejoribeiro/my-dev-environment/blob/main/dot_config/.config/nvim/lua/plugins.lua

**Comment by ray-x** [comment]
Did you do anything else other than typing? I mean switch active windows back and forth etc.

**Comment by felipejoribeiro** [comment]
no. The only thing I did, for the message to appear, was press the shortcut to activate the floating window. Then the red text appeared and the floating window showed the behavior.

**Comment by ray-x** [comment]
I feel you are using an old version of lsp_signature. Can you update it to the latest?
I do not see any validated code in init.lua line 479.

**Comment by felipejoribeiro** [comment]
i updated, my head is in:
![image](https://user-images.githubusercontent.com/39244876/197186238-a91ff92d-2dda-4c76-9749-f19aa042dff2.png)

The problem persists.
